### DISCUSSION TITLE
Having the secondary y axis

### DISCUSSION THREAD

**Discussion by Elisa-Lee** [PHPOffice/PhpSpreadsheet#3649] [Ideas]
Due to different scale on numbers, a secondary y-axis will be helpful to show the value. Like to know the clicks on web page could be some thousands, but the purchasing on this page may be only 30+. To see them both in one diagram, a secondary y-axis is needed. 

In Excel combo diagram, it's possible to add the line on secondary axis. Why not to have it?

**Comment by oleibman** [comment]
We have greatly expanded the functionality of charts over the past year or so. I have not yet been able to figure out how to implement secondary axes. You are not the first to suggest this feature. It is useful to know that there are others who want it.
